<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cont_gui</class>
 <widget class="QWidget" name="cont_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>504</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>823</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_layout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Transmission</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="port_settings_bt">
            <property name="text">
             <string>Port settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="test_bt">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="quit_bt">
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Device:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="dev_sel_sb">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dev_get_info_bt">
            <property name="text">
             <string>Get status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dev_set_time_bt">
            <property name="text">
             <string>Set time</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="log_text">
          <property name="font">
           <font>
            <family>Monospace</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QProgressBar" name="prog_bar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="log_cb">
            <property name="text">
             <string>Log </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="loghex_cb">
            <property name="text">
             <string>Hex</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Device:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Typ:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="dev_type_l">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Date &amp; Time:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateTimeEdit" name="dev_dt">
                <property name="date">
                 <date>
                  <year>2020</year>
                  <month>1</month>
                  <day>1</day>
                 </date>
                </property>
                <property name="maximumDateTime">
                 <datetime>
                  <hour>23</hour>
                  <minute>59</minute>
                  <second>59</second>
                  <year>2083</year>
                  <month>12</month>
                  <day>31</day>
                 </datetime>
                </property>
                <property name="minimumDateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>2020</year>
                  <month>1</month>
                  <day>1</day>
                 </datetime>
                </property>
                <property name="currentSection">
                 <enum>QDateTimeEdit::DaySection</enum>
                </property>
                <property name="displayFormat">
                 <string>dd.MM.yyyy HH:mm:ss</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="dev_time_now_bt">
                <property name="text">
                 <string>&lt;-  Set time to &quot;Now&quot;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Parameter(s):     Addr:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="dev_par_addr_sb">
                <property name="maximum">
                 <number>255</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Values:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="dev_par_vals_le">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhDialableCharactersOnly|Qt::ImhDigitsOnly</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="dev_par_set_bt">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Set</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="dev_ow_search_bt">
                <property name="text">
                 <string>Search 1-wire</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="dev_ow_addrs_cb"/>
              </item>
              <item>
               <widget class="QPushButton" name="dev_ow_temperature_bt">
                <property name="text">
                 <string>Temperature</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>port_settings_bt</tabstop>
  <tabstop>test_bt</tabstop>
  <tabstop>log_text</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>log_cb</tabstop>
  <tabstop>loghex_cb</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>log_text</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quit_bt</sender>
   <signal>clicked()</signal>
   <receiver>cont_gui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>891</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
